scripts/check-db-dump-is-decryptable.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+LATEST_DUMP=$(aws s3 ls s3://digitalmarketplace-database-backups | sort | tail -n 1 | awk '{print $4}')
```

Comment:
We generate the file name for `S3_POST_URL_DATA`, is there a way for us to store it and use that? It would allow us to use the script to check any backup as well as making sure we're checking the thing we've just created and not some other file that happens to have a later date for some reason.